spec/refinement/used_path_spec.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.describe Refinement::UsedPath do
```

Comment:
This seems empty, is just WIP or not needed or intentionally setup for future tests to be added?

Reply:
I intend to add tests (before merging), but since these are mostly model objects, there shouldn't be all that much interesting in the tests